pkg/sql/codegen_alps.go

```diff
@@ -670,22 +672,22 @@ func (meta *metadata) getDenseColumnInfo(keys []string, refColumns map[string]*c
 			shape[0] = len(fields)
 			if userSpec, ok := refColumns[ct.Name()]; ok {
 				output[ct.Name()] = &columns.ColumnSpec{
-					ct.Name(),
-					false,
-					shape,
-					userSpec.DType,
-					userSpec.Delimiter,
-					nil,
-					*meta.featureMap}
```

Comment:
Is there a linter complaining we need to explicitly name the field name? If so, please file the change in a separate PR.

Reply:
The go vet complains: `[govet] xxx(struct) composite literal uses unkeyed fields [E]` while I'm editing in another file.
I found that there was not much code to be changed, so I changed them conveniently.